diff -durN bins-1.1.7.orig/bins-edit-gui bins-1.1.7/bins-edit-gui
--- bins-1.1.7.orig/bins-edit-gui	Fri May 24 11:38:52 2002
+++ bins-1.1.7/bins-edit-gui	Fri May 24 11:39:41 2002
@@ -115,7 +115,7 @@
 Gnome->init("bins-edit-gui", $version);
 Gtk->init;
 
-$glade = "/usr/local/share/bins/bins-edit-gui.glade";
+$glade = "/usr/X11R6/share/bins/bins-edit-gui.glade";
 if (! -r $glade) {
     $glade = "bins-edit-gui.glade" ; # developer hack
     print "DEVELOPER HACK\n";
